tests/s3/copy_object/test_copy_obj_simultaneous_del.py

```diff
@@ -219,3 +276,51 @@ def test_45525(self):
                                                         is_mpu=True)
         LOGGER.info("ENDED: Test Delete source object when copy multipart object is in "
                     "progress in same bucket")
+
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_object_copy
+    @pytest.mark.tags("TEST-45526")
+    @CTFailOn(error_handler)
+    def test_45526(self):
+        """Test Delete destination object when copy multipart object is in progress
+        in same bucket"""
+        LOGGER.info("STARTED: Test Delete destination object when copy multipart object is in "
+                    "progress in same bucket")
+        self.create_put_parallel_copy_and_delete_object((self.srcbuck, "src-obj", self.srcbuck,
+                                                         "dest-obj", "NoSuchKey", 5,
+                                                         self.file_path), (self.srcbuck,
+                                                         "dest-obj"),
+                                                        is_mpu=True)
+        LOGGER.info("ENDED: Test Delete destination object when copy multipart object is in "
+                    "progress in same bucket")
+
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_object_copy
+    @pytest.mark.tags("TEST-45527")
+    @CTFailOn(error_handler)
+    def test_45527(self):
+        """Test Delete destination multipart copied object with the same name as source object
+        in the same bucket"""
+        LOGGER.info("STARTED: Test Delete destination multipart copied object with the same name "
+                    "as source object in the same bucket")
+        self.create_put_parallel_copy_and_delete_object((self.srcbuck, "obj1", self.srcbuck,
```

Comment:
Are we performing all permutations combinations written in Expected Results section from [TDS](https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/1079640340/TDS+F-20A+S3+Copy+Object+API)? At least from code I don't think so.